syntax/lean.vim

```diff
@@ -0,0 +1,113 @@
+" Vim syntax file
```

Comment:
Maybe it's worth considering `git subtree`'ing lean.vim rather than copy pasting it if we're going to conflict with it?

Then we at least can pull in any changes automatically

Reply:
I'm happy to archive lean.vim and move all development here.  git subtree is really inconvenient for development though (i.e. you need to make changes in lean.nvim, then commit them to lean.vim, then update lean.nvim again).

Reply:
Ok that sounds even better.